src/main/java/frc/trigon/robot/subsystems/collector/CollectorConstants.java

```diff
@@ -0,0 +1,28 @@
+package frc.trigon.robot.subsystems.collector;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+
+public class CollectorConstants {
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(1).per(Units.Second.of(1)),
+            Units.Volts.of(7),
+            null,
+            null
+    );
+
+    public enum CollectorState {
+        RESTING(0, Rotation2d.fromDegrees(50)),
+        COLLECTING(5, Rotation2d.fromDegrees(30)),
+        OPENING(2, Rotation2d.fromDegrees(40));
```

Comment:
For resting, the power should be 0 and the angle in degrees should be 90. 
For collecting, the power should be negative(?) and with an angle of 0. 
Opening should have the same angle as the collecting, just without power.